src/socket/unix_socket.cr

```diff
@@ -13,31 +13,38 @@
 # ```
 class UNIXSocket < Socket
   getter path : String?
+  getter? abstract : Bool
 
-  # Connects a named UNIX socket, bound to a filesystem pathname.
-  def initialize(@path : String, type : Type = Type::STREAM)
+  # Connects a named UNIX socket.
+  #
+  # A non-abstract socket is bound to a filesystem pathname.
```

Comment:
Keep the original comment, but add a note to the `UNIXSocket` class about `abstract` (as for `UNIXServer`).